tests/CppUTestExt/MockSupportTest.cpp

```diff
@@ -909,6 +939,388 @@ TEST(MockSupportTest, customObjectWithFunctionComparatorThatFailsCoversValueToSt
     CHECK_EXPECTED_MOCK_FAILURE_LOCATION(failure, __FILE__, __LINE__);
 }
 
+TEST(MockSupportTest, customTypeOutputParameterSucceeds)
+{
```

Comment:
Ok. Multiple inheritance is a *potential* problem for old or embedded compilers, so would be good if we can look for a different solution here :( Sorry about that.